python/ray/dataframe/dataframe.py

```diff
@@ -1207,7 +1211,9 @@ def __setitem__(self, key, value):
         raise NotImplementedError("Not Yet implemented.")
 
     def __len__(self):
-        raise NotImplementedError("Not Yet implemented.")
+        return sum(
```

Comment:
For lengths, we have a new field called `_lengths` which is the length of each partition. You can simply use `sum(_lenghts)` for this one and it should be reasonably fast.

Reply:
resolved by calling sum(_lengths)